src/docs/reference/modules/Migrations/README.md

```diff
@@ -0,0 +1,81 @@
+# Data Migrations (`OrchardCore.Data.Migration.DataMigration`)
+
+Data Migration classes can be used to alter the content type definitions (like by adding new __types__, or configuring their __parts__ and __fields__), 
+initializing recipes or creating indices.
+
+There should be one Migrations file per module inheriting from __DataMigration__. 
```

Comment:
If a module has more than one feature, and multiple migrations per feature each migration should be decorated with the `Feature` attribute

i.e. `[Feature("OrchardCore.ContentFields.Indexing.SQL")]`